screens/BooksScreen.js

```diff
@@ -21,28 +21,69 @@ export default function BooksScreen() {
     getBooks();
   }, []);
 
-  const authorName = data.map((e) => {
-    e.authors.map((author) => {
-      console.log(author.name)
-      return author.name
-    })
-     });
+  //looking for ISBN
+  const availabilities =  data.map((book) => {
+    return !!book && book.availability
+  });
+
+  const isbnArray = availabilities.map((availability)=>{
+    console.log("availability.isbn:", availability !== undefined ? availability.isbn : '')
+    return availability !== undefined ? availability.isbn : ''
+  })
+    console.log("isbnArray:", isbnArray)
+
+  const isbn = isbnArray.forEach((isbn) => {
+    console.log("ISBN:",  typeof isbn === "string" ? isbn : '')
+    return typeof isbn === "string" ? `[ISBN: ${isbn}]` : ''
+  })
+  console.log("constISBN:", isbn)
+
+  const renderItem = ({ item }) => {
+
+    const authors = item.authors.length > 1
+      ? `[authors: ${item.authors[0].name} and ${item.authors.length - 1} more]`
+      : `[author: ${item.authors[0].name}]`
+      // console.log(item.authors.length)
+
+    return (
+      <View>
+        <Text style={styles.item}>
+          {item.title} {authors} {isbn}
+        </Text>
+
+        {/* {isbnArray.forEach(isbn =>
+          <Text>
+            {typeof isbn === "string" ? isbn : '' }
+          </Text>
+        )} */}
```

Comment:
because const isbn it's not what I need to pass to my list . 
I tried to loop trough isbn directly in my renderItem function. 
No success isbn still not visible , nothing happens.

Reply:
see in [this discussion](https://github.com/SoleCincis/tracker-App/pull/33/files#r740356746)